SneakPeek/res/layout/sneaker_category_listitem.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical" >
+    
+	<TextView 
+	    android:id="@+id/txtCategoryName"
+	    android:layout_width="wrap_content"
+	    android:layout_height="wrap_content"
+	    android:textStyle="bold"
+	    android:text="@string/category_name"
+	    android:padding="@dimen/list_item_padding"
+	    />
+
+	<TextView
+	    android:id="@+id/txtCategoryDesc"
+	    android:layout_width="wrap_content"
+	    android:layout_height="wrap_content"
+	    android:layout_alignParentLeft="true"
+	    android:layout_alignParentTop="true"
```

Comment:
By default, this view will align to its parent left and top, so I think you don't need this. I think it might be overlapping with the txtCategoryName view because it is not put where this view should be relative to the aforementioned view.